### PR TITLE
Add dynamicEnable property

### LINKED ISSUE #28: New property: dynamicEnable
See ScratchAddons/ScratchAddons#1858

Desc is the same as `dynamicDisable`, but just for enabling.

### PR DESCRIPTION
**Pull request by Hans5958**
Merge after ScratchAddons/ScratchAddons#1961

Resolves #28

### CODE REVIEW

**Comment by TheColaber** [comment]
Pretty sure this will also apply to userscripts too as well as persistent scripts with both enable and disable.

**Comment by Hans5958** [comment]
> Pretty sure this will also apply to userscripts too as well as persistent scripts with both enable and disable.

Sorry? `dynamicEnable` doesn't mind the other properties.

File: 1/1.8.json [hunk 1]
```diff
@@ -464,6 +464,11 @@
 			"type": "boolean",
 			"description": "Determines whether the userstyles could be disabled dynamically.",
 			"default": false
+		},
+		"dynamicEnable": {
+			"type": "boolean",
+			"description": "Determines whether the userstyles could be enabled dynamically.",
```
Comment (TheColaber) [inline 1/1.8.json:470]: whether the *addon*

**Comment by Hans5958** [comment]
Will be replaced with a direct commit. Stand by.

**Pull request closed without merging**